pkg/storage/allocator_test.go

```diff
@@ -1913,6 +1924,79 @@ func TestAllocatorComputeAction(t *testing.T) {
 	}
 }
 
+func TestAllocatorComputeActionDisableStatsRebalance(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	ctx := context.Background()
+	stopper, g, _, a, _ := createTestAllocator( /* deterministic */ false)
+	defer stopper.Stop(ctx)
+	stores := []*roachpb.StoreDescriptor{
```

Comment:
A comment explaining why the different constants (the `RangeCount` values, the `us-east` constraint, etc.) and what this test is testing would be very helpful for anyone who has to debug it in the future.